test/test_files/terrainbox/terrainBox.inp

```diff
@@ -0,0 +1,63 @@
+#¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨¨#
```

Comment:
I think this case/file should be named `terrain_box`

Reply:
Agreed that this should be named terrain_box. But more importantly, the input file must have the same name as the directory in order for it to run with ctest. When you update the directory and input file names, make sure you update the CMakeLists.txt too

Reply:
Fixed.